test/OpenTelemetry.Tests/Trace/Sampling/SamplerBuilderTests.cs

```diff
@@ -0,0 +1,114 @@
+// <copyright file="SamplerBuilderTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Moq;
+using Xunit;
+
+namespace OpenTelemetry.Trace.Tests;
+
+public class SamplerBuilderTests : IDisposable
+{
+    public SamplerBuilderTests()
+    {
+        Environment.SetEnvironmentVariable(OTelEnvSamplerDetector.OTelTracesSamplerEnvVarKey, null);
+    }
+
+    public void Dispose()
+    {
+        Environment.SetEnvironmentVariable(OTelEnvSamplerDetector.OTelTracesSamplerEnvVarKey, null);
+
+        GC.SuppressFinalize(this);
```

Comment:
Why? We do not have any finalizer. This line can be removed.

Reply:
You know, ctrl+c... ;)

Fixed: 7a4caacf093efc155fe24f02dc5a590dbb26cc32